modules/custom/cu_core/cu_core.install

```diff
@@ -340,8 +340,36 @@ function cu_core_requirements($phase) {
   $requirements = array();
   if ($phase === 'runtime') {
     $t = get_t();
+    $overriddenFeatures = array();
+    // This is a way to hard code (almost never the best?) features we know will be overridden.
+    // Just put the sting name of the feature in.
+    $featureStatesToIgnore = [
+      'cu_sitewide',
+      'cu_sitemap',
+      'cu_feature_callout',
+      'cu_social_links',
+      'cu_webform',
+      'cu_mega_menu',
+      'cu_ldap',
+      'other features to overlook...'
```

Comment:
Remove this line

Reply:
I am assuming you are referring to only line 354.